ddtrace/internal/debug.py

```diff
@@ -23,6 +24,10 @@
 
 logger = get_logger(__name__)
 
+# The architecture function spawns the file subprocess on the interpreter
+# executable. We make sure we call this once and cache the result.
+architecture = callonce(platform.architecture)
```

Comment:
my bad, I forgot `platform.architecture` takes arguments 🙁 
```suggestion
architecture = callonce(lambda: platform.architecture())
```